app/javascript/crayons/Buttons/__stories__/Buttons.stories.jsx

```diff
@@ -0,0 +1,67 @@
+import { h } from 'preact';
+import ButtonsDoc from './Buttons.mdx';
+import { ButtonNew } from '@crayons';
+import SampleIcon from '@img/cog.svg';
+
+export default {
+  component: ButtonNew,
+  title: 'BETA/Buttons',
+  parameters: {
+    docs: {
+      page: ButtonsDoc,
+    },
+  },
+  argTypes: {
+    primary: {
+      description:
+        'This prop defines whether or not your button will have *primary* style (in practice: high contrast, filled with accent color). Keep in mind ideally there should be only one primary button per entire component, or sometimes even per entire view.',
+      table: {
+        defaultValue: { summary: false },
+      },
+    },
+    rounded: {
+      description:
+        'By enabling this setting you can turn all corners of the button to be fully rounded. We usually use it **only** for buttons containing **icon without label**.',
+      table: {
+        defaultValue: { summary: false },
+      },
+    },
+    destructive: {
+      description:
+        'For various destructive actions we can have special styling for button which will add red-ish coloring.',
```

Comment:
Right now AFAIR we only have one, in dashboard - for deleting posts.. It's displayed against white background so the contrast ratio isn't really an issue there. Since this destructive style button is very rare and it's usage will rather always be against white background I think we could leave it as is. But I'm also 100% fine with darkening our red variable to improve the contrast ratio against that grey background. What do you think?

Reply:
Damn accessibility! :D Anyway, I'll update our red shade.

Reply:
Is this still todo in this PR or do you want to handle this separately?

Reply:
Whooops! Yea, I'll fix it in this PR as well 👍

Reply:
Done.